src/modules/navigator/rtl_params.c

```diff
@@ -59,9 +60,9 @@ PARAM_DEFINE_FLOAT(RTL_RETURN_ALT, 60);
 
 
 /**
- * Return mode loiter altitude
+ * Return mode loiter altitude (relative to home).
  *
- * Stay at this altitude above home position after RTL descending.
+ * Stay at this altitude above home position after return to home mode descending.
```

Comment:
This should be:

> Descend to this altitude (above home position) after return, and wait for time defined in RTL_LAND_DELAY.

Note, arguably we might need to revisit everything again after rally points supported. Will altitude be above home position or the rally point ground level? I presume the former is possible/easy but the later is desirable :-)